### DISCUSSION TITLE
Looking for Help

### DISCUSSION THREAD

**Discussion by theafricaninternetgroup** [elsa-workflows/elsa-core#3585] [General]
I am trying get upto speed with understanding Elsa my project requires the use of it.Occasionally I am getting  stuck,I am looking for sort of mentor to help through  my journey of course at a fee.If there anyone able to help I would really  appreciate.

Regards

**Comment by underwater** [comment]
I am in a similar situation, and was considering if we can get a few people together who may be willing to pay for learning, perhaps we can get someone with some experience to do a workshop / online course for us....

**Reply by theafricaninternetgroup** [reply]
Great idea @sfmskywalker what do think?

**Comment by michaelhaddad4** [comment]
Hi I am also looking for Elsa workflow skills. I need an experienced elsa workflow programmer to help us get started. Any ideas on where i can find this person will be appreciated.
Thank you

**Comment by sfmskywalker** [comment]
@theafricaninternetgroup @underwater @michaelhaddad4 I love the idea. Perhaps, to make the most of everyone's time and money, you guys let me know what topics and scenarios you'd like to learn more about so that I can do some prep work.
Alternatively, we could have a free form group call and tackle whatever comes up.

**Reply by michaelhaddad4** [reply]
Hi @sfmskywalker My colleague and I need to integrate workflows with a web application (c#, .Net, SQL Server etc). We are considering Elsa and would love to know more and see if we can practically get it working with a simple workflow. I am the solution architect and my colleague is the developer who will be doing the work. We are happy to join in with others. We are UTC +2 time zone.

**Reply by db-seko** [reply]
Hi @sfmskywalker - is there any update on the group call? As my colleague @michaelhaddad4 has mentioned above, we would like to see more and get to understand what Elsa can do for us.

**Comment by underwater** [comment]
I would also be working within a AspNetCore backend typically to
orchestrate a a multi step process (getting approval, sending a
notification, uploading a document etc).   I am in CET+1 (Paris time)
@michaelhaddad4 have you already looked through the V2 documentation
<https://elsa-workflows.github.io/elsa-core/docs/next/quickstarts/quickstarts-console-hello-world>,
with the examples?   I know the V3 will be out (in the coming months?) and
API may be different, so not sure, if we should wait to train on the V3 project,
@sfmskywalker do u think the V2 / V3 API is sufficiently close to train on V2?
first ?

**Comment by theafricaninternetgroup** [comment]
@sfmskywalker  thanks for your offer,I am developing a blazor Saas HR solution I would like to use elsa for customer onboarding this is basically a multi step long running workflow.If the customer does not finish the onboarding process in one go.I would like elsa to take the customer/tenant to their last step. I am in  CET+1

**Reply by underwater** [reply]
@theafricaninternetgroup and @michaelhaddad4  may I ask what training budget you are willing to spend on these sessions ?  On my side it is only a few hundred euros.... Perhaps we should post this on the discord to see if other persons would be interested in a few structured sessions.

**Reply by michaelhaddad4** [reply]
Good morning all. There are two of us from our side (myself and @db-seko). I will be mainly an observer and architecture question poser, whilst my colleague will want the tech details. We are also prepared to pay a few hundred euros for the sessions with the option of asking for an hour or so later for specific advice for which we expect to pay an hourly rate. Looks like we are getting there. @sfmskywalker please can you suggest a way forward if you are available and amenable.

**Reply by theafricaninternetgroup** [reply]
@underwater 
My budget is about USD200.00

**Comment by dodyg** [comment]
I will start creating samples for Esla as soon as 3.0 previews become available. I maintain over 1500 samples for ASP.NET Core  https://github.com/dodyg/practical-aspnetcore

**Reply by underwater** [reply]
That would be indeed so awesome ;-)

**Reply by sfmskywalker** [reply]
That's quite an impressive repository 💪🏻

**Comment by underwater** [comment]
Hi @theafricaninternetgroup, I understand,
Let us keep in mind this is a niche framework, and I believe would would
need to assemble a larger of interested persons to make it worth it
financially for the mentor.  Typically such expertise is around several
hundred dollars per hour.

**Reply by theafricaninternetgroup** [reply]
@underwater I get you so how much would sufficient to motivate to conduct  this course I will  sponsor  this course  may be one or two people  who have expressed  can subsize  the course.Can we talk

**Comment by underwater** [comment]
Well I estimate contributors who understand the framework well would have a typical européen consulting day rate (about 250 - 300  per hour).   I suppose if each of us who are in it for learning, and are willing to pay only a few hundred each out of pocket as personal expense, we'd better try to assemble at least 6 students to make it somewhat manageable for a contributor to organize a few sessions with Q&A.   Of course, I am just guessing, and would leave contributors who are willing to teach speak for themselves, but I am just saying this so that what we propose makes sense also financially.

**Reply by michaelhaddad4** [reply]
Hi Can I just ask for clarity on rates please. Are you saying Euros250 per hour, Euros2000 per day? Whilst this may be a going rate from consulting houses we have never paid that much for independent consultants. An ex-colleague now managing an IT firm in the UK says they charge UKP1000 per day. I hope we can do something for say Euros 1000 in total. Thanks

**Comment by underwater** [comment]
I honestly don't know, I am just making a guess as to what a mentor will be
charging for this type of thing.
Of course someone can do it for less just to share knowledge, but as a
transaction, it would be common to pay 250 euro per hour for this sort of
training in Europe.
Where are you guys based ?

**Reply by michaelhaddad4** [reply]
Hi @theafricaninternetgroup my colleague @db-seko and I are in South Africa.

**Reply by theafricaninternetgroup** [reply]
@michaelhaddad4 I am in Zambia,basically in the same time zone.

**Reply by dodyg** [reply]
Hello everyone from Cairo up north.

**Reply by michaelhaddad4** [reply]
Hi @dodyg . I see that you maintain hundreds of samples and thus I assume you know a lot about Elsa. Are you maybe able to assist us in our efforts to understand more about Elsa from both a technical and architecture point of view?

**Reply by theafricaninternetgroup** [reply]
How many are we who are willing to take this training and how much are we willing to pay in total so that we can quickly find a mentor.This thread has been on for sometime  we should  now move to the next stage

**Reply by dodyg** [reply]
@michaelhaddad4 our expertise is in other commercial workflow (OptimaJet) but we are looking at ELSA 3 really seriously for adoption.

**Comment by sfmskywalker** [comment]
Hi everyone, thanks for all the input. After having gone through your comments, here's what I am offering:

- My standard rate as an independent consultant is EUR 120 ex VAT. 
- Before we do the actual workshop, I propose that we start with a free-form, free-of-charge, Q&A group call. This would allow me to not only answer basic questions, but also to take notes and collect topics that need further elaboration as input for preparing subsequent workshops.
- If you allow me to record the workshops (except the initial Q&A) and use them as I please, I will do them free of charge.

Please let me know your thoughts. Next Monday between 10:00 and 15:00 UTC+1 would be a good window for me to do the Q&A.

**Reply by dodyg** [reply]
How about Noon on that day (Jan 23rd)

**Reply by michaelhaddad4** [reply]
@sfmskywalker Sipke that is a fantastic offer thank you very much. We are in any time that suits everyone. I happily accept that you can record and re-use.

**Reply by michaelhaddad4** [reply]
Hi Sipke @sfmskywalker 
Once again thanks for the offer. Are we on for Monday 23rd? Will you send an invite?

**Reply by theafricaninternetgroup** [reply]
@michaelhaddad4 What time on 23rd January and now that we have the hourly  rate from @sfmskywalker How many of are willing team up for this the paid sessions

**Reply by theafricaninternetgroup** [reply]
> How about Noon on that day (Jan 23rd)

Which time zone is this ?

**Reply by sfmskywalker** [reply]
Hi @michaelhaddad4, I wanted to wait for @theafricaninternetgroup to let us know if they'd be available too, since they started the discussion. 

@theafricaninternetgroup I am in the UTC+1 timezone, but perhaps it's best if we talk about times in UTC, given the different participating timezones.

With regards to the sessions, I am offering to do them for free if everyone is OK with me recording them. If anyone prefers for the sessions to not be recorded, please let me know.

Is everyone OK with Monday, January 23, 11:00-13:00 UTC? 
It will be a free-form Q&A. Some questions I might be able to answer readily, while others will enable me to get a better understanding of what workshops I should prepare.

**Reply by theafricaninternetgroup** [reply]
Thank you.Looking forward  to Monday.I will wait for invite/link

**Reply by michaelhaddad4** [reply]
Fantastic thank you. 11am UTC or even GMT for those of us in the ex british colonies.
My colleague @db-seko will also attend. Do you need email addresses for the meet?

**Reply by theafricaninternetgroup** [reply]
@sfmskywalker when are you sharing the link for tomorrow's Q&A session

**Reply by sfmskywalker** [reply]
Yes, please send me your email addresses and I’ll send out a teams link today. See you tomorrow!

**Comment by underwater** [comment]
I regrettably won't manage to participate after all, but wish u a good
training sessions.

**Comment by michaelhaddad4** [comment]
Thanks Sipke
I am ***@***.******@***.***> and my colleague is
***@***.******@***.***>

See you tomorrow.

**Reply by theafricaninternetgroup** [reply]
dennis.mwape@africaninternetgroup.com

**Reply by sfmskywalker** [reply]
@michaelhaddad4 Looks like your mailserver or GitHub redacted your email addresses. Please let me know your addresses here, or send a private message to sipkeschoorstra@outlook.com and I will add you to the list.

**Reply by sfmskywalker** [reply]
@dodyg I don't think I received your email address, but if you would like to join the Monday session, please let me know and you'll be invited as well.

**Reply by sfmskywalker** [reply]
@underwater Sorry you can't make it. Hopefully some other time then. With regards to your question about V2 and V3, they are both quite different and are essentially different products. Some concepts are shared, but API-wise, totally different.

**Reply by dodyg** [reply]
Hi @sfmskywalker ,

Please add 

dodyg@silverkeytech.com
sherifr@silverkeytech.com

**Comment by sfmskywalker** [comment]
Hi everyone, I'm running late by about 5 minutes. See you soon!

**Comment by sfmskywalker** [comment]
https://v3.elsaworkflows.io/
https://github.com/elsa-workflows/elsa-documentation

**Reply by theafricaninternetgroup** [reply]
@sfmskywalker 
Great session looking forward  to the next hands on  sessions 
Thanks alot

**Reply by dodyg** [reply]
Fantastic session. Thank you @sfmskywalker.

**Reply by michaelhaddad4** [reply]
Thanks again @sfmskywalker .
Good luck with Release 3.
I look forward to our workshops.

**Comment by theafricaninternetgroup** [comment]
@sfmskywalker 
Hello,I have sent my specific requirement of the UI driven Workflow to your outlook inbox.I anxiously await to hear form you soon.

Thanks
